src/containers/shared/components/Transaction/SetHook/utils.ts

```diff
@@ -0,0 +1,133 @@
+import { zeroPad } from '../../../transactionUtils'
+
+const TRANSACTION_TYPES: Record<string, number> = {
```

Comment:
This might get annoying to update every time a new transaction is added. I exported from the xrpl.js the TRANSACTION_TYPE_MAP. Might think about that. Should also add the URIToken transactions.

Reply:
We're working on some tree-shaking from ripple-binary-codec, given the size of the package I think we'll need to wait on that first.

Reply:
ripple-binary-codec needs to be cleaned up a bit before it can be used in the Explorer without introducing a lot of bloat